po/common/fr.po

```diff
@@ -4764,6 +4764,10 @@ msgctxt "attribute_ecoscore_grade_title"
 msgid "Eco-Score %s"
 msgstr "Eco-Score %s"
 
+msgctxt "attribute_ecoscore_a_plus_description_short"
```

Comment:
Do you have those strings in common.pot ?

Reply:
I verified, they exists, see https://github.com/openfoodfacts/openfoodfacts-server/blob/ecoscore2/po/common/common.pot